freeplane/src/main/java/org/freeplane/view/swing/map/overview/MapOverviewUtils.java

```diff
@@ -0,0 +1,100 @@
+package org.freeplane.view.swing.map.overview;
+
+import java.awt.Color;
+import java.awt.Insets;
+import java.awt.Rectangle;
+
+import javax.swing.JScrollBar;
+import javax.swing.JScrollPane;
+
+import org.freeplane.core.resources.ResourceController;
+import org.freeplane.view.swing.map.MapView;
+
+public class MapOverviewUtils {
+    static Color complementaryColor(final Color color) {
+        return new Color(0xFF - color.getRed(), 0xFF - color.getGreen(), 0xFF - color.getBlue());
+    }
+
+    private static MapOverviewAttachPoint getMapOverviewAttachPoint() {
+        ResourceController resourceController = ResourceController.getResourceController();
+        String rawAttachPoint = resourceController.getProperty(MapOverviewConstants.PROP_MAP_OVERVIEW_ATTACH_POINT,
+                MapOverviewAttachPoint.SOUTH_EAST.name());
+        try {
+            return MapOverviewAttachPoint.valueOf(rawAttachPoint);
+        } catch (IllegalArgumentException e) {
+            return MapOverviewAttachPoint.SOUTH_EAST;
+        }
+    }
+
+    public static void convertOriginByAttachPoint(MapView mapView, Rectangle bounds) {
```

Comment:
I think part of the best practices is putting public methods ahead of the private methods they call so that when someone reads a class its purpose comes first and implementation details follow.

Because this method is used only by classes related to MapViewPane I would move it to MapViewPane and remove MapView as its parameter. 

ResizePanelMouseHandler could be given a reference to MapViewPane or obtain it from the event. It does not need to know MapView directly.